src/instructlab/model/serve.py

```diff
@@ -196,6 +196,7 @@ def serve(
             vllm_args=vllm_args,
             host=host,
             port=port,
+            vllm_path=ctx.obj.config.serve.vllm.vllm_path,
```

Comment:
@russellb would you want vllm_path to be a flag for the `ilab model serve` command too?